examples/index.json

```diff
@@ -1075,8 +1075,32 @@
         "column": 3,
         "category": "Experimental features",
         "samples": [{
-                "name": "Table Queries",
-                "url": "table-queries"
+                "name": "Select all the fields",
+                "url": "table-queries-select-all"
+            },
+            {
+                "name": "Select a few field for the output",
+                "url": "table-queries-select-with-few-output-fields"
+            },
+            {
+                "name": "Select with filtering",
+                "url": "table-queries-select-with-filtering"
+            },
+            {
+                "name": "Select with order-by clause",
+                "url": "table-queries-select-with-order-by"
+            },
+            {
+                "name": "Select with group-by clause",
+                "url": "table-queries-select-with-group-by"
+            },
+            {
+                "name": "Joining two tables",
```

Comment:
Should this and the following start with "Join" instead of "Joining" for consistency?